File: ARCADE/Intro/Level 9_3: longestDigitsPrefix.cs
    using System;
     
    public class Test
    {
    	public static string longestDigitsPrefix(string inputString){
            string result = "";
            for (int i = 0; i < inputString.Length; i++){
                if ('0' <= inputString[i] && inputString[i] <= '9'){
                    result += inputString[i];
                }
                else break;
            }
            return result;
        }
    	public static void Main()
    	{
    		string inputString="123aa1";
    		Console.WriteLine(longestDigitsPrefix(inputString));
    	}
    }

File: ARCADE/Intro/Level 9_3: longestDigitsPrefix.pas
program longestDigitsPrefix;
 
function longestDigitsPrefix(inputString: string): string;
var rs : string;
    i : integer;
begin
    rs := '';
    for i := 1 to length(inputString) do
        if((48 <= ord(inputString[i])) and (ord(inputString[i]) <= 57)) then
            rs := rs + inputString[i]
        else break;
    longestDigitsPrefix := rs;
end;
 
var inputString : string;
begin
	inputString := '123aa1';
	writeln(longestDigitsPrefix(inputString));
end.
